EIPS/eip-2400.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 2400
+title: Transaction Receipt URI
```

Comment:
Following EIP-681 and EIP-1710, perhaps a better title is "URL Format for Transaction Receipts".

Or following EIP-831 and EIP-1328, "URI Format for Transaction Receipts".

Reply:
agreed on that. Changed the wording and used URL.